### PR TITLE
Nodeup clean

### PR DESCRIPTION
**Pull request by gambol99**
Cleaning up elements in the nodeup command. Separated into logical commits to make reviewing eaiser
- https://github.com/kubernetes/kops/commit/6188ca559f17492695eb49a007f6d33aba1c7b68 removes the duplicates methods
- https://github.com/kubernetes/kops/commit/90676da61377fed78bca2a250d8a11578dd569c8 cleans up some imports, comments and formatting etc
- https://github.com/kubernetes/kops/pull/5352/commits/2256a2c588c417374c773261664097979690e9cb fixing up the issues highlighted in the review

### CODE REVIEW

File: nodeup/pkg/model/directories.go [hunk 1]
```diff
@@ -29,11 +29,12 @@ type DirectoryBuilder struct {
 
 var _ fi.ModelBuilder = &DirectoryBuilder{}
 
+// Build is ensures an number of directories
```
Comment (KashifSaadat) [inline nodeup/pkg/model/directories.go]: Bit confusing, did you mean something like: `Build ensures the Kubernetes bin directory is mounted and files executable on any changes`?

File: nodeup/pkg/model/directories.go [hunk 2]
```diff
@@ -42,8 +43,7 @@ func (b *DirectoryBuilder) Build(c *fi.ModelBuilderContext) error {
 				{"/bin/mount", "--bind", "/home/kubernetes/bin", "/home/kubernetes/bin"},
```
Comment (KashifSaadat) [inline nodeup/pkg/model/directories.go]: Whilst you're in this file, could maybe substitute the paths here for `dir` variable defined above.

File: nodeup/pkg/model/network.go [hunk 3]
```diff
@@ -31,9 +31,11 @@ type NetworkBuilder struct {
 
 var _ fi.ModelBuilder = &NetworkBuilder{}
 
+// Build is responsible for configuring the network cni
 func (b *NetworkBuilder) Build(c *fi.ModelBuilderContext) error {
 	var assetNames []string
 
+	// @TODO need to clean up the code section isn't not easier to read
```
Comment (KashifSaadat) [inline nodeup/pkg/model/network.go]: `isn't not` whaaat? :P

File: nodeup/pkg/model/packages.go [hunk 4]
```diff
@@ -29,6 +30,7 @@ type PackagesBuilder struct {
 
 var _ fi.ModelBuilder = &DockerBuilder{}
 
+// Build is responsible for installing packahes
```
Comment (KashifSaadat) [inline nodeup/pkg/model/packages.go]: `packahes` => `packages`

**Comment by KashifSaadat** [comment]
The cleanup all looks great, just minor comment updates to do. I'll leave open for a second pair of eyes to review, but LGTM.

File: nodeup/pkg/model/directories.go (hunk 2 above)
Comment (gambol99) [reply nodeup/pkg/model/directories.go]: good shout!

File: nodeup/pkg/model/network.go (hunk 3 above)
Comment (gambol99) [reply nodeup/pkg/model/network.go]: somewhat ironic given its trying to say it isn't the easiest to read :-)

**Comment by gambol99** [comment]
fixed up the issues highlighted ... much appreciated @KashifSaadat

**Pull request merged**